src/domains/Lotto/index.js

```diff
@@ -0,0 +1,43 @@
+import {
+  MAX_LOTTO_NUMBER_COUNT,
+  MAX_LOTTO_NUMBER,
+  MIN_LOTTO_NUMBER,
+  ERROR_MESSAGES,
+} from "./constants/index.js";
+
+export class Lotto {
+  #numbers;
+
+  constructor(numbers) {
```

Comment:
상수를 왜 분리해야 하는걸까요?
```suggestion
import {
  MAX_LOTTO_NUMBER_COUNT,
  MAX_LOTTO_NUMBER,
  MIN_LOTTO_NUMBER,
  ERROR_MESSAGES,
} from "./constants/index.js";

export class Lotto {
  static Range  = { MIN: 1, MAX: 45 }
  static SIZE = 6
  
  #numbers;

  constructor(numbers) {
```
이렇게 선언해놓고
```tsx
Lotto.Range.MIN
Lotto.Range.MAX
Lotto.SIZE
```
이렇게 Lotto에 묶어서 사용할 수 있어요 ㅎㅎ